### ISSUE TITLE
Merging Lora fine-tuned models with MoE

### ISSUE THREAD

**Issue by AmineBechar07** [arcee-ai/mergekit#426]
Hello everyone, I have an issue, is there a way to use finetuned models Lora adapters (safetensors) on my own data for merging? Is there someone who managed to realize it?

**Comment by flash-rad-ai** [comment]
Hello @AmineBechar07 were you able to achieve this?

**Comment by AmineBechar96** [comment]
Yes. I merged Lora with basemodels. and then made the finetuning
